src/components/marker-chart/Canvas.js

```diff
@@ -343,11 +387,12 @@ class MarkerChartCanvas extends React.PureComponent<Props, State> {
 
     ctx.fillStyle = '#fff';
     ctx.fillRect(
-      TIMELINE_MARGIN_LEFT,
-      rowIndex * rowHeight - viewportTop + 1, // Add plus one for borders.
-      containerWidth - TIMELINE_MARGIN_LEFT,
-      rowHeight - 2 // Subtract 2 for borders.
+      0,
```

Comment:
The former dimensions did not cover the left-most labels and would leave a highlighted border on top therefore changed.

Reply:
The `+1` and `-2` gave a gap between the top border and highlighted region
![issue#2727_L390_1](https://user-images.githubusercontent.com/26107248/91328876-1c818a00-e7e5-11ea-8860-71a5d45b55ce.png)

And without `+1` and keeping `-2` we get 
![issue#2727_L390_2](https://user-images.githubusercontent.com/26107248/91329055-52bf0980-e7e5-11ea-8fe8-f27f9a6a5610.png)

I found removing the `+1` and doing `-2` as `-1` as perfect as it was filling out the gaps (but might've been covering the border? But I couldn't tell)
This 1px gap might not be visible in larger screens but in the smaller screens this difference is quite visible